### PR TITLE
fix: Credit link changes from X to LinkedIn for Angela Angelini

### LINKED ISSUE #7231: Ang_ngl is no longer on Twitter.
Currently, according to the website, credit for our lovely mascot goes to Angela Angelini, however the Twitter account linked no longer exists.

Ref: https://twitter.com/nodejs/status/1760358555180220811?lang=en
Ref: https://nodejs.org/en/about/branding

### PR DESCRIPTION
**Pull request by TenzDelek**
## Description

Currently, according to the website, credit for our lovely mascot goes to Angela Angelini, however the Twitter account linked no longer exists.
## Validation
image

## Related Issues
Fixes #7231 

### Check List

- [x] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [x] I have run `npm run format` to ensure the code follows the style guide.
- [x] I have run `npm run test` to check if all tests are passing.
- [x] I have run `npx turbo build` to check if the website builds without errors.
- [ ] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

File: apps/site/pages/en/about/branding.mdx [hunk 1]
```diff
@@ -9,7 +9,7 @@ Please review the [trademark policy](https://trademark-policy.openjsf.org/) for
 
 ## Node.js® Mascot
 
-Credit to [@Ang_ngl on X](https://x.com/Ang_ngl) for designing and contributing the Rocket Turtle.
+Credit to [@angeliningl on LinkedIn](https://www.linkedin.com/in/angeliningl/) for designing and contributing the Rocket Turtle.
```
Comment (avivkeller) [inline apps/site/pages/en/about/branding.mdx]:
```suggestion
Credit to [Angela Angelini](https://www.linkedin.com/in/angeliningl/) for designing and contributing the Rocket Turtle.
```

IMO we can just use the name

Comment (TenzDelek) [reply apps/site/pages/en/about/branding.mdx]: sure

**Review by avivkeller — APPROVED** [review]

**Review by AugustinMauroy — DISMISSED** [review]
@micheleriva  who's close to her, you'd be able to validate that she's OK with us putting her linkedin on the site.

**Comment by micheleriva** [comment]
> @micheleriva who's close to her, you'd be able to validate that she's OK with us putting her linkedin on the site.

Will tell you ASAP

**Comment by micheleriva** [comment]
@AugustinMauroy LinkedIn is fine. Thank you so much for asking!

**Review by AugustinMauroy — APPROVED** [review]

**Comment by avivkeller** [comment]
Thanks for your contribution!

**Pull request merged**